doc/denite.txt

```diff
@@ -142,19 +142,21 @@ EXAMPLES						*denite-examples*
 
 	" Ack command on grep source
 	call denite#custom#var('grep', 'command', ['ack'])
-	call denite#custom#var('grep', 'recursive_opts', [])
-	call denite#custom#var('grep', 'final_opts', [])
-	call denite#custom#var('grep', 'separator', [])
 	call denite#custom#var('grep', 'default_opts',
 			\ ['--ackrc', $HOME.'/.ackrc', '-H',
 			\ '--nopager', '--nocolor', '--nogroup', '--column'])
-	" Ripgrep command on grep source
-	call denite#custom#var('grep', 'command', ['rg'])
 	call denite#custom#var('grep', 'recursive_opts', [])
-	call denite#custom#var('grep', 'final_opts', [])
+	call denite#custom#var('grep', 'pattern_opt', ['--match'])
```

Comment:
Ability to specify multiple patterns. Ability to specify patterns starting with `-`.